config/StoredProcedures/AssociateProductCustomer.sql

```diff
@@ -0,0 +1,14 @@
+
+use imp_db_dev;
+DROP PROCEDURE IF EXISTS AssociateProductCustomer;
+
+DELIMITER $$
+CREATE PROCEDURE AssociateProductCustomer
+(IN _ProductName INT, IN _CustomerName INT)
```

Comment:
Might want to rename these as _ProductID and _CustomerID. Also want to change 'int' to 'int unsigned'